docs/config.yaml

```
@@ -151,6 +151,10 @@ clouds:
     # OPTIONAL: accounts to share newly created AMIs with
     grant_users:
       - "1234567890"
+    # OPTIONAL: Add labels of a user-defined key and value to attach to the AMI. 
+    # Tags help you manage, identify, organize, search for, and filter resources.
```

It's OK not to explain AWS tags here. Maybe just

```suggestion
    # OPTIONAL: AWS tags to add on the AMIs
```

?

---

Sure. Updated it.